ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>This is test 4.11, please add a test for 4.10 to test the following error.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
This following error is from the Copley drive (not the CCW). You could not do this unless you intentionally break the Copley drive hardware (and we may need to send it back to vendor to fix it). Therefore, it is better to use the Simulink model for this test only.